setup.py

```diff
@@ -13,13 +13,14 @@ def read(*parts):
     with codecs.open(os.path.join(HERE, *parts), "rb", "utf-8") as f:
         return f.read()
 
-# Sets the __version__ variable
-__version__ = None
-exec(open('dispaset/_version.py').read())
+# Retrieve the release tag as fallback version
+exec(open(os.path.join(HERE, 'dispaset/_release.py')).read().strip())
```

Comment:
why don't we read based on the function above instead of doing exec(). it can be a txt file instead of py

Reply:
Because I'm lazy :-). Seriously, I prefer use import at runtime (`__init__.py`) and avoid to have to read a file. During the build, I don't mind to do exec().

Perhaps, if you agree we can use [setuptools_scm_git_archive](https://github.com/pypa/setuptools_scm#notable-plugins) plugin to remove the `_release` file and let dispaset exits when users not use a tagged tarball.

```python
try:
    from ._version import version
except ImportError:
    import sys
    sys.exit("Failed to find (autogenerated) _version.py. This might be due to"
             "using not tagged GitHub's tarballs. Either use release tarball"
             "or clone repository from GitHub via git.")

```

Reply:
yeah i guess it is ok to exec() on build. 
but it seems very punishing to exit for a version check! As I said some non-tech savvy users just download a zip from github